code/modules/clothing/head/misc.dm

```diff
@@ -428,3 +428,14 @@
 	desc = "An extra-mustahabb way of showing your devotion to Allah."
 	icon_state = "taqiyahred"
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/small
+
+/obj/item/clothing/head/shrine_wig
+	name = "shrine maiden's wig"
+	desc = "Purify in style!"
+	flags_inv = HIDEHAIR //bald
+	mob_overlay_icon = 'icons/mob/large-worn-icons/64x64/head.dmi'

```

Comment:
this is copied code from cage helmet and you havent even changed the icon

Reply:
It is copied, but only because it's too big a sprite for the normal head slot. large-worn-icons contains more than just the cage helmet; the file has been changed to contain the wig.

Reply:
i mean its just 1 tile bigger and theres a clear place on where you can shorten it but ok